atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -319,8 +321,9 @@ def run_image_task(self, image_build_conf):
                 task_result = self.session.getTaskResult(task_id)
             except Exception as exc:
                 task_result = repr(exc)
-            raise RuntimeError('image task, {}, failed: {}'
-                               .format(task_id, task_result))
+            raise RuntimeError('image task failed. see {}/taskinfo?taskID={}    \n'
```

Comment:
Consider using triple quotes to build a multi-line string:
```python
from textwrap import dedent

raise RuntimeError(dedent("""\
    image task failed
    see ....
    underlying task failure: ...""".format(...)
))
```

`dedent` will remove the leading spaces added. This makes the source code look properly indented while not affecting the string created.